app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,10 @@
   </head>
 
   <body>
+    <nav class="row">
+      <%= link_to "Pets", "/pets", class: "col-2 nav-link" %>
+      <%= link_to "Shelters", "/shelters", class: "col-2 nav-link" %>
+    </nav>
```

Comment:
Navigation added to give users links to shelters and pets from anywhere on the site.